src/makeCalculator.js

```diff
@@ -4,7 +4,31 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {};
+
+  calculator.result = 0;
+
+  calculator.operate = (callback, number) => {
+    calculator.result = callback(number);
+
+    return calculator;
+  };
+
+  calculator.add = (a) => calculator.result + a;
+
+  calculator.subtract = (a) => calculator.result - a;
```

Comment:
Code style issue: You should use the `-=` operator to subtract the number from the `result`. Currently, you are using the `-` operator, which is not correct. The `-` operator simply subtracts two numbers, it does not modify the `result`. The `-=` operator, on the other hand, subtracts the number from the `result` and assigns the result to the `result`. So, you should change your code to `calculator.result -= a;`.